### PR TITLE
Update UserGuide.md

### PR DESCRIPTION
**Pull request by kuanjr**
Updates userguide for edit and delete command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -196,17 +196,35 @@ Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [i/INFO]
 
 * Edits the client named `NAME`.
   * `Name` is case-insensitive. E.g. `John` will match `john`.
+  * Spaces in `NAME` will be counted as separate names, unless `NAME` fully matches client's name
+    * Example: `edit John Doe p\88888888` will find clients with names containing `John` and `Doe`,
+                there exists a client with the name `John Doe`
   * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`.
-  * Only full words will be matched e.g. `Han` will not match `Hans`.
```
Comment (AD-NAP) [inline docs/UserGuide.md:200]: I think this line is still a valid point to be in the UG

File: docs/UserGuide.md [hunk 2]
```diff
@@ -196,17 +196,35 @@ Format: `edit NAME [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [s/SALARY] [i/INFO]
...
+
 * `DATE` has to be in the format **YYYY-MM-DD**.
 * **At least one** of the optional fields must be provided.
 * Existing values will be updated with the input values.
 * When editing tags, the existing tags of the client will be removed i.e adding of tags is not cumulative.
 * You can remove all the client’s tags by typing `t/` without specifying any tags after it.
 * You can remove the client's info by typing `i/` without specifying any info after it.
 
-Example:
-* `edit n/John Doe d/2020-12-04` Edits the previous meeting date of the client with the name `John Doe` to `2020-12-04` which is 4th Dec 2020.
```
Comment (AD-NAP) [inline docs/UserGuide.md:209]: Maybe example should be kept to keep the format consistent?

File: docs/UserGuide.md (hunk 1 above)
Comment (kuanjr) [reply docs/UserGuide.md:200]: Ok added it back in

File: docs/UserGuide.md (hunk 2 above)
Comment (kuanjr) [reply docs/UserGuide.md:209]: Ok added it back in as a separate scenario

**Review by AD-NAP — APPROVED** [review]
LGTM!

**Pull request merged**